src/System.Security.Permissions/src/System/Security/Permissions/DataProtectionPermissionAttribute.cs

```diff
@@ -0,0 +1,20 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Security.Permissions
+{
+    [AttributeUsage(AttributeTargets.Method | AttributeTargets.Constructor | AttributeTargets.Class
+     | AttributeTargets.Struct | AttributeTargets.Assembly, AllowMultiple = true, Inherited = false )] 
+    [System.Security.Permissions.HostProtection(MayLeakOnAbort = true)]
```

Comment:
You can remove all these `[System.Security.Permissions.HostProtection(MayLeakOnAbort = true)]` .. we don't implement this in coreclr.